cumulusci/tasks/salesforce/custom_settings_wait.py

```diff
@@ -78,17 +77,16 @@ def _poll_action(self):
 
         self.record = None
         for row in query_results["records"]:
-            setupOwnerId = str(row["SetupOwnerId"])
-            if setupOwnerId.startswith("00D"):
+            if row["SetupOwnerId"].startswith("00D"):
                 self.record = row
 
-        if not self.record:
-            raise SalesforceException(
-                "Hierarchical Custom Settings Org Default record not found"
+        if self.record:
+            self.poll_complete = not self._poll_again()
+        else:
+            self.logger.info(
```

Comment:
This line doesn't have test coverage, so I suspect we could use a test for the case where the settings object exists but no record was found.